web/src/features/modals/SettingsModal.tsx

```diff
@@ -90,10 +90,9 @@ export function SettingsModalContent() {
         {__('tooltips.selectLanguage')}
       </Button>
       {isLanguageSelectorOpen && (
-        <LanguageSelector
-          className="top-[185px] left-auto right-auto z-10 mt-4 w-60 overflow-x-hidden shadow-lg sm:top-[200px]"
-          setLanguageSelectorOpen={setIsLanguageSelectorOpen}
-        />
+        <div className="top-[185px] left-auto right-auto z-10 mt-4 w-60 overflow-x-hidden shadow-lg sm:top-[200px]">
+          <LanguageSelector />
+        </div>
```

Comment:
We should also add the theme selector here and make sure it looks good on mobile :)